nix/auto-luks.nix

```diff
@@ -67,6 +67,24 @@ with utils;
               luksFormat</command>.
             '';
           };
+
+          mountPoint = mkOption {
+            default = false;
+            type = types.nullOr types.string;
+            description = ''
+              This option is required so the autoLuks module knows where the
```

Comment:
If required, why allow "null"?

Reply:
That was meant as an escape hatch for people that know what they are doing. There might be cases where it is being used without using it for a filesystem.

Reply:
I changed the type signature make it explicit that I want either a `str` or the string "none". While technically not required I think making the `"none"` case explicit in the type serves as a good way to document the existence of it.